--- a/pre-build.sh	2017-12-03 22:41:41.956247591 +0100
+++ b/pre-build.sh	2017-12-03 22:45:27.516375927 +0100
@@ -38,8 +38,3 @@
     | sort -u \
     | awk -F'[()]' '{print "#define "$2" amd"$2" //"$0}'\
     >> include/rename_symbol.h
-
-FW_DIR="/lib/firmware/$KERNELVER"
-mkdir -p $FW_DIR
-cp -ar /usr/src/amdgpu-17.40-492261/firmware/radeon $FW_DIR
-cp -ar /usr/src/amdgpu-17.40-492261/firmware/amdgpu $FW_DIR
